src/resources/locales/en.json

```diff
@@ -488,37 +488,37 @@
         "theme": {
             "auto": "Automatic",
             "dark": "Dark",
-            "description": "The application theme corresponds to the theme of your operating system.",
+            "description": "Application theme matches operating system.",
             "light": "Light",
             "title": "Application Theme"
         }
     },
     "wizard": {
         "buttons": {
             "discover": "Discover what's new",
-            "goToBooks": "Go to my books",
+            "goToBooks": "Go to my publications",
             "next": "Next"
         },
         "description": {
             "annotations": "Highlight important passages, take notes. Thanks to the advanced mode you can even make annotations on the fly.",
-            "catalogs": "Borrow books from your local library or discover new books with the OPDS catalogs -- It works like an online catalog and allow you to access a variety of publications from multiple sources.",
+            "catalogs": "Borrow publications from your local library or discover publications via OPDS feeds. Online catalogs allow you to access a variety of publications from multiple sources.",
             "home": "Thorium gets a makeover, let's discover what's new!",
             "readingView1": "Thorium adapts itself to your reading preferences: choose amongst various themes, adjust text parameters or use readaloud while folllowing the text in the screen.",
             "readingView2": "Search, navigate to a precise location, add bookmarks or enjoy the full screen mode for distraction-free reading sessions.",
-            "yourBooks": "Welcome in your personnal bookshelf: filter and easily find a book in the grid view or do a advanced search in the table view."
+            "yourBooks": "Welcome to your personnal bookshelf: filter and easily find a publication in the grid view or do a advanced search in the table view."
```

Comment:
I think there is a typo there on "personnal" -> "personal". I would write "easily filter and find a publication in the grid view, or perform an advanced search in the table view"